lib/sidekiq/cron.rb

```diff
@@ -21,6 +21,19 @@ class Configuration
 
       def initialize
         @default_namespace = 'default'
+        @strict_cron_parsing = false
+      end
+
+      # Throws an error when a natural language cron string would get
+      # parsed into multiple cron lines. By default the `fugit` gem is
+      # permissive when parsing natural language cron strings. Only the first
+      # cron line is used in that case and all other ones are ignored.
+      def strict_cron_parsing!
```

Comment:
Maybe we can use an attr_accessor instead of a bang method 🤔?

Reply:
I have no strong preference between the two. Whichever you prefer.

Reply:
I think it will be more consistent with other option flags that may arise in the future.